WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/Services/ComputerService.cs

```diff
@@ -60,9 +63,10 @@ public class ComputerService : WorkerStreamWrapper
 
     public override async Task<Output> Process(ITaskHandler taskHandler)
     {
-      using var scopedLog = Logger.BeginNamedScope("Execute task",
-                                                   ("Session", taskHandler.SessionId),
-                                                   ("TaskId", taskHandler.TaskId));
+      using var scopedLog = LoggerExt.BeginNamedScope(Logger,
```

Comment:
sooooo ugly : it is an extension method of ILogger. Should work with Logger.BeginNamedScope